sdk/netapp/arm-netapp/samples/v19/javascript/README.md

```diff
@@ -1,200 +0,0 @@
-| [volumesRevertRelocationSample.js][volumesrevertrelocationsample]                                 | Reverts the volume relocation process, cleans up the new volume and starts using the former-existing volume. x-ms-original-file: specification/netapp/resource-manager/Microsoft.NetApp/stable/2023-05-01/examples/Volumes_RevertRelocation.json                                                                                                             |
-| [volumesRevertSample.js][volumesrevertsample]                                                     | Revert a volume to the snapshot specified in the body x-ms-original-file: specification/netapp/resource-manager/Microsoft.NetApp/stable/2023-05-01/examples/Volumes_Revert.json                                                                                                                                                                              |
-| [volumesUpdateSample.js][volumesupdatesample]                                                     | Patch the specified volume x-ms-original-file: specification/netapp/resource-manager/Microsoft.NetApp/stable/2023-05-01/examples/Volumes_Update.json                                                                                                                                                                                                         |
-
-## Prerequisites
-
-The sample programs are compatible with [LTS versions of Node.js](https://github.com/nodejs/release#release-schedule).
-
-You need [an Azure subscription][freesub] to run these sample programs.
-
-Samples retrieve credentials to access the service endpoint from environment variables. Alternatively, edit the source code to include the appropriate credentials. See each individual sample for details on which environment variables/credentials it requires to function.
-
-Adapting the samples to run in the browser may require some additional consideration. For details, please see the [package README][package].
-
-## Setup
-
-To run the samples using the published version of the package:
-
-1. Install the dependencies using `npm`:
-
-```bash
-npm install
-```
-
-2. Edit the file `sample.env`, adding the correct credentials to access the Azure service and run the samples. Then rename the file from `sample.env` to just `.env`. The sample programs will read this file automatically.
-
-3. Run whichever samples you like (note that some samples may require additional setup, see the table above):
-
-```bash
-node accountsCreateOrUpdateSample.js
-```
-
-Alternatively, run a single sample with the correct environment variables set (setting up the `.env` file is not required if you do this), for example (cross-platform):
-
-```bash
-npx cross-env NETAPP_SUBSCRIPTION_ID="<netapp subscription id>" NETAPP_RESOURCE_GROUP="<netapp resource group>" NETAPP_SUBSCRIPTION_ID="<netapp subscription id>" NETAPP_RESOURCE_GROUP="<netapp resource group>" node accountsCreateOrUpdateSample.js
```

Comment:
this file has this issue but the version is not the latest one so we just remove them.